lms/djangoapps/certificates/models.py

```diff
@@ -294,9 +294,9 @@ def get_certificate_generation_candidates(self):
                 [CertificateStatuses.readable_statuses.get(status, "") for status in statuses]
             )
 
-        # If statuses_to_regenerate is not present in task_input then, certificate generation task was run to
-        # generate certificates for white listed students
-        return "for exceptions"
+        # If students is present in task_input then, certificate generation task was run to
+        # generate certificates for white listed students otherwise it is for all students.
+        return _("for exceptions") if 'students' in task_input_json else _("All learners")
```

Comment:
This also needs a translators note to clear up context.

Further, the grammar/display seems odd here; one branch returns a phrase with a leading capital letter (`All learners`), while the other does not (`for exceptions`). Regardless of the greater context, it is unlikely that this is correct.